umh-core/Makefile

```diff
@@ -27,19 +41,6 @@ S6_OVERLAY_VERSION = 3.2.0.2
 BENTHOS_UMH_VERSION = 0.6.2
 REDPANDA_VERSION = 24.3.8
 
```

Comment:
this PR is likely broken because of commits, why is it removed here? please rebase your commit upon [eng-2773-action-deploy-dataflowcomponent](https://github.com/united-manufacturing-hub/united-manufacturing-hub/tree/eng-2773-action-deploy-dataflowcomponent) branch